.travis.yml

```diff
@@ -21,14 +28,52 @@ matrix:
         env: DJANGO='django>=2.0,<2.1.0' TESTDB=sqlite
       - python: "2.7"
         env: DJANGO='django>=2.0,<2.1.0' TESTDB=postgres
+      - python: "2.7"
+        env: DJANGO='django>=2.1,<2.2.0' TESTDB=sqlite
+      - python: "2.7"
+        env: DJANGO='django>=2.1,<2.2.0' TESTDB=postgres
+      - python: "2.7"
+        env: DJANGO='django>=2.2,<3.0' TESTDB=sqlite
+      - python: "2.7"
+        env: DJANGO='django>=2.2,<3.0' TESTDB=postgres
+      - python: "2.7"
+        env: DJANGO='django>=3.0,<3.1.0' TESTDB=sqlite
+      - python: "2.7"
+        env: DJANGO='django>=3.0,<3.1.0' TESTDB=postgres
       - python: "2.7"
         env: DJANGO='https://github.com/django/django/archive/master.tar.gz' TESTDB=sqlite
       - python: "2.7"
         env: DJANGO='https://github.com/django/django/archive/master.tar.gz' TESTDB=postgres
+      - python: "3.5"
+        env: DJANGO='django>=3.0,<3.1.0' TESTDB=sqlite
+      - python: "3.5"
+        env: DJANGO='django>=3.0,<3.1.0' TESTDB=postgres
+      - python: "3.5"
+        env: DJANGO='https://github.com/django/django/archive/master.tar.gz' TESTDB=sqlite
+      - python: "3.5"
+        env: DJANGO='https://github.com/django/django/archive/master.tar.gz' TESTDB=postgres
+      - python: "3.8"
+        env: DJANGO='django>=1.11.0,<1.12.0' TESTDB=sqlite
+      - python: "3.8"
+        env: DJANGO='django>=1.11.0,<1.12.0' TESTDB=postgres
+      - python: "3.8"
+        env: DJANGO='django>=2.0,<2.1.0' TESTDB=sqlite
+      - python: "3.8"
+        env: DJANGO='django>=2.0,<2.1.0' TESTDB=postgres
+      - python: "3.8"
+        env: DJANGO='django>=2.1,<2.2.0' TESTDB=sqlite
+      - python: "3.8"
+        env: DJANGO='django>=2.1,<2.2.0' TESTDB=postgres
```

Comment:
I can't help wondering if these work, even if Django doesn't explicitly support it. I don't see any commits that claim to be related to 3.8 support, in their git HEAD.

Reply:
While it's not officially supported, several projects run their tests on Django 2.1 and Python 3.8, so it's currently doable to at least some extent.